llvm/utils/git/code-format-helper.py

```diff
@@ -126,13 +126,18 @@ def should_be_excluded(self, path: str) -> bool:
             return True
         return False
 
+    def should_include_extensionless_file(self, path: str) -> bool:
+        return path.startswith('libcxx/include')
```

Comment:
```suggestion
        return path.startswith("libcxx/include")
```
To make the formatter happy.